Plugins/BTCPayServer.RockstarDev.Plugins.Payroll/BTCPayServer.RockstarDev.Plugins.Payroll.csproj

```diff
@@ -21,7 +21,7 @@
 	</PropertyGroup>
 
 	<!-- This will make sure that referencing BTCPayServer doesn't put any artifact in the published directory -->
-	<ItemDefinitionGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
I don't remember why I put that here, but it is probably because it makes things easier to debug or maybe fix ef migrations... unsure.